.github/workflows/pull_request.yml

```diff
@@ -85,19 +82,16 @@ jobs:
     name: 🐪 Check Perl
     needs: build_backend
     runs-on: ubuntu-latest
-    if: github.event_name == 'pull_request'
+    if: github.event_name == 'pull_request' && !contains(github.event.pull_request.changed_files, '.md')
     steps:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 1
     - name: Fetch origin/main
-      # we need origin/main to have comparison linting work
       run: |
         git remote set-branches --add origin main
         git fetch --no-tags --prune --progress --no-recurse-submodules --depth=5 origin main
     - name: Restore taxonomies dates
```

Comment:
```suggestion
    - name: Restore taxonomies dates
      # here we first restore dates from git for taxonomies to avoid build them all
      # see https://stackoverflow.com/a/60984318/2886726
```